src/ucp/tag/offload.c

```diff
@@ -558,7 +558,7 @@ ucs_status_t ucp_tag_offload_sw_rndv(uct_pending_req_t *self)
     packed_len   = ucp_tag_rndv_rts_pack(rndv_rts_hdr, req);
 
     status = uct_ep_tag_rndv_request(ep->uct_eps[req->send.lane],
-                                     req->send.msg_proto.tag.tag,
+                                     req->send.msg_proto.tag,
```

Comment:
There are multiple issues as a result of reducing the line in this file.
(also highlighted by code format checker)

Reply:
fixed